nixos/modules/programs/apeloader.nix

```diff
@@ -0,0 +1,23 @@
+{ config, lib, pkgs, ... }:
+
+let
+  cfg = config.programs.apeloader;
+in
+{
+  options.programs.apeloader.enable = lib.mkEnableOption
+    "support for αcτµαlly pδrταblε εxεcµταblε binaries";
```

Comment:
is there a reason to use non-latin alphabet here?

Reply:
Yes, it is an upstream idiosyncrasy to spell "actually portable executable" using visually similar characters from the Greek alphabet. That said if there is a reason to diverge from upstream we can also undo that.

Reply:
I think it can break screen readers and reduce accessibility (because it will be read by nixos-search, etc.).